src/components/transactions/message-view/ClaimRewardsTxDetails.vue

```diff
@@ -341,7 +341,7 @@ export default {
 .reponsiveClaimValidator {
   margin: 0 2rem;
 }
-@media screen and (max-width: 1035px) {
+@media screen and (max-width: 822px) {
```

Comment:
@jbibla this seems hacky. shouldn't breakpoints be standardized to just a couple?

Reply:
yes

Reply:
Thank you for the remark. This code was actually not needed and just deleted it.

Let's get luniehq\lunie-api#493 and this merged! 🚀 💃